frontend/www/js/omegaup/components/course/AssignmentDetails.vue

```diff
@@ -326,6 +327,10 @@ export default class CourseAssignmentDetails extends Vue {
   onSubmit(): void {
     this.$emit('submit', this, this.scheduledProblemList?.problems ?? []);
   }
+
+  onChangeSelect(event: { target: { value: string } }): void {
```

Comment:
```suggestion
  onChangeSelect(event: Event): void {
```

este es el tipo correcto para [`change` event](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event)